packages/framework-core/src/services/graphql/graphql-generator.ts

```diff
@@ -20,7 +21,18 @@ export class GraphQLGenerator {
   private readonly subscriptionGenerator: GraphQLSubscriptionGenerator
   private readonly typeInformer: GraphQLTypeInformer
 
-  public constructor(
+  private static singleton: GraphQLGenerator | undefined
+  
+  public static getInstance(config: BoosterConfig, logger: Logger): GraphQLGenerator {
```

Comment:
This is great, the only thing I'd change is that I think we're using the word `build` instead of `getInstance` in other singletons in the codebase, so I'd suggest to rename it. What do other @boostercloud/booster-core members think about this?

```suggestion
  public static build(config: BoosterConfig, logger: Logger): GraphQLGenerator {
```

Reply:
Agree